spec/helpers/find/subject_helper_spec.rb

```diff
@@ -58,23 +58,23 @@
           let(:subjects) { [find_or_create(:secondary_subject, :biology)] }
 
           it 'returns the correct financial information' do
-            expect(subject.first.financial_info).to be_nil
+            expect(subject.first.financial_info).to eq('Bursaries available')
           end
         end
 
         context 'with scholarship only' do
           let(:subjects) { [find_or_create(:secondary_subject, subject_name: 'made up subject', scholarship: 1_000_000_000)] }
 
           it 'returns the correct financial information' do
-            expect(subject.first.financial_info).to be_nil
+            expect(subject.first.financial_info).to eq('Scholarships available')
           end
         end
 
         context 'with scholarship and bursary' do
           let(:subjects) { [find_or_create(:secondary_subject, :chemistry)] }
 
           it 'returns the correct financial information' do
-            expect(subject.first.financial_info).to be_nil
+            expect(subject.first.financial_info).to eq('Scholarships and bursaries are available')
```

Comment:
This needs to be 'or' rather than 'and' because they can't have both.

Reply:
Cool - have updated. Just to flag this will affect Find as it is now too.